packages/react/src/patterns/sub-patterns/PictogramItem/PictogramItem.js

```diff
@@ -0,0 +1,72 @@
+/**
+ * Copyright IBM Corp. 2016, 2018
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import ContentItem from '../ContentItem/ContentItem';
+import {
+  settings as ddsSettings,
+  // markdownToHtml,
+} from '@carbon/ibmdotcom-utilities';
+//   import { ArrowRight20 } from '@carbon/icons-react';
+//   import { LinkWithIcon } from '../../../components/LinkWithIcon';
+import PropTypes from 'prop-types';
+import React from 'react';
+import { settings } from 'carbon-components';
+const { stablePrefix } = ddsSettings;
+const { prefix } = settings;
+
+/**
+ * Converts markdown to html and returns object to be used in `dangerouslySetInnerHTML`
+ *
+ * @param {string} string string to be converted to html
+ * @returns {object} object with a __html key containing converted string
+ */
+//   const convertMarkdown = string => ({
+//     __html: markdownToHtml(string, {
+//       bold: false,
+//     }),
+//   });
```

Comment:
I'm sure you know this stuff and other comments will need to go?

Reply:
I've cleaned the comments up